libr/core/core.c

```diff
@@ -73,7 +73,7 @@ static int on_fcn_delete (void *_anal, void* _user, RAnalFunction *fcn) {
 	return 0;
 }
 
-static int on_fcn_rename(void *_anal, void* _user, RAnalFunction *fcn, const char *oname) {
+static int on_fcn_rename(void* _user, RAnalFunction *fcn, const char *oname) {
 	RCore *core = (RCore*)_user;
```

Comment:
anal != core this change is wrong

Reply:
It would break tests if this were wrong. no one uses the first argument.